nibabel/testing/__init__.py

```diff
@@ -21,8 +21,10 @@
 try:
     from numpy.testing import dec
     skipif = dec.skipif
+    slow = dec.slow
 except ImportError:
-    from numpy.testing.decorators import skipif
+    from numpy.testing.decorators import (skipif, slow)
```

Comment:
Something's weird with the testing because this is never getting hit, even on the numpy==1.7.1 build. See 2007.5, here:

https://codecov.io/gh/nipy/nibabel/commit/0c36551c7398dcfb132fe4ce37b115c23d183b72/build

I know it's not really relevant to this PR, but any thoughts? I checked the [travis log](https://api.travis-ci.org/v3/job/503684117/log.txt), and see no indication that anything other than 1.7.1 was installed.

Reply:
No immediate idea, I am away from the laptop ATM, but if I were with it I would remove try/except and see how it fails

Reply:
Ah, it's because `numpy.testing.dec` has existed [forever](https://github.com/numpy/numpy/blob/v1.7.1/numpy/testing/__init__.py#L10). It's just an alias of an imported module, and the module it's aliasing has changed from `numpy.testing.decorators` to [`numpy.testing._private.decorators`](https://github.com/numpy/numpy/blob/master/numpy/testing/__init__.py#L13).

How about we just drop the `try`/`except` bit, and load `dec`, `skipif`, and `slow`?